packages/server/tests/acceptance/rpc_batch3.spec.ts

```diff
@@ -1893,6 +1893,7 @@ describe('@api-batch-3 RPC Server Acceptance Tests', function () {
 
     it('Should return a batch of requests', async function () {
       const testAccount = await servicesNode.createAliasAccount(100, relay.provider, Utils.generateRequestId());
+      await new Promise((r) => setTimeout(r, 5000));
```

Comment:
Not a fan of sleeps, what's this for?

Reply:
Do you need to poll for the existence of the account instead?

Reply:
we've added a bunch of those delays across the testing suites to ensure enough time is allowed for the mirror node to pick up the changes, I belive a polling mechanism is a better approach.

Also not a big fan of `sleeps` but is better to wait a `5s` delay and have the tests succeed, rather that waste around `10m` of testing due to the lack of.

If we go the route of doing a polling, instead of wait, it should be worked as a helper tool that can be reutilized across the testing suites, and still have a `MAX` amount of wait while polling, and then a follow-up PR for implementing it and refactoring all the acceptance testing suites that currently use the `sleep` approach.

Reply:
We can create a separate task to implement polling as Alfredo suggested. So if you agree, we can close this PR and work separately on this.

Reply:
I agree that the polling mechanism should be in another task, as it will increase the scope of this change and it's unrelated.